src/components/Input/Input.stories.tsx

```diff
@@ -26,6 +26,13 @@ Default.args = {
   label: 'Input Label'
 };
 
+export const WithMask = Template.bind({});
+WithMask.args = {
+  placeholder: 'MM / DD / YYYY',
+  label: 'Input Date for Mask',
+  mask: '99/99/9999'
```

Comment:
@viniciusaa should this value be displayed in the input upon user interaction?

Reply:
No, this value determines that the input will accept only numbers and the "/" will be permanent and non-editable (check the documentation that I have added).